README.md

```diff
@@ -54,20 +54,33 @@ This meta-role requires a permission policy similar to the following:
 
 * [Basic usage](https://github.com/cisagov/cert-read-role-tf-module/tree/develop/examples/basic_usage)
 
+## Requirements ##
+
+| Name | Version |
+|------|---------|
+| terraform | ~> 0.12.0 |
+| aws | ~> 2.0 |
+
+## Providers ##
+
+| Name | Version |
+|------|---------|
+| aws | ~> 2.0 |
+
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role. | list(string) | [] | no |
-| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | string | | yes |
-| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_bucket_path>/site.example.com/*. | string | "live" | no |
-| hostname | The FQDN corresponding to the certificate to be read (e.g. site.example.com). | string | | yes |
+| account_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
+| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored | `string` | n/a | yes |
+| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_path>/site.example.com/* | `string` | "live" | no |
```

Comment:
I think `"live"` should be `` `live` ``? That's what the command I've been using outputs (I dug that up on Chat). Command I've been using for reference:
``terraform-docs markdown --no-escape --no-header . | sed -E 's/`"|"`/`/g' ``

Reply:
Ah thanks- I haven't been using `terraform-docs` lately, but I will go back to it now.  

See https://github.com/cisagov/cert-read-role-tf-module/pull/18/commits/f13d1a4ac20e1d269620a0794928bccd9de0b79f.

Reply:
Ah, ok. I found it the easiest way to update since I was touching a lot of older repos.